apps/web/src/pages/templates/components/templates-store/ChannelNode.tsx

```diff
@@ -0,0 +1,30 @@
+import { NodeProps, Handle, Position, getOutgoers, useReactFlow } from 'react-flow-renderer';
+import styled from '@emotion/styled';
+
+import { NodeStep } from '../../../../components/workflow';
+
+const NodeStepStyled = styled(NodeStep)`
+  width: 200px;
+`;
+
+export const ChannelNode = ({ id, data }: NodeProps) => {
+  const { getNode, getEdges, getNodes } = useReactFlow();
+  const thisNode = getNode(id);
+  const isParent = thisNode ? getOutgoers(thisNode, getNodes(), getEdges()).length : false;
+  const noChildStyle = isParent ? {} : { border: 'none', background: 'transparent' };
+
+  return (
+    <NodeStepStyled
+      data={data}
+      Icon={data.Icon}
+      Handlers={() => {
+        return (
+          <>
+            <Handle type="target" id="b" position={Position.Top} />
+            <Handle style={noChildStyle} type="source" id="a" position={Position.Bottom} />
```

Comment:
Would be a problem if multiple components share the same id? Even if being part of a parent component? Thinking in the reconciliation algorithm and when debugging.

Reply:
I'm not sure if this is even needed, but I've copied it from the other place 🥲
This `id` is not used as the HTML el `id` but as the internal library `handleId`.
It shouldn't have any impact on the reconciliation.